numpy/lib/histograms.py

```diff
@@ -792,8 +795,10 @@ def histogramdd(sample, bins=10, range=None, normed=False, weights=None):
 
     range : sequence, optional
         A sequence of lower and upper bin edges to be used if the edges are
-        not given explicitly in `bins`. Defaults to the minimum and maximum
-        values along each dimension.
+        not given explicitly in `bins`. An entry of None in the sequence
```

Comment:
So they must match the dimensions in some way?

Reply:
Yep, should be a tuple of length `D`. Will update.